### DISCUSSION TITLE
After upgrade from 3.3.12 to 3.3.15 selected node is not reloaded

### DISCUSSION THREAD

**Discussion by Sega100500** [vakata/jstree#2725] [Q&A]
After upgrade from jsTree 3.3.12 to 3.3.15 selected node is not reloaded

1. tree is working fine
2. select any node in tree
3. when select (click mouse) any other node all work fine
4. but when I try select seleted node - node is not reloaded (not fire any events)

Why? May be any options help?

**Comment by Tipetan** [comment]
Hi,
I have the same issue!
Did anybody have a solution?

**Comment by Sega100500** [comment]
@Tipetan 
Not a very big difference in versions, so far I've downgrade small part of code to same as v3.3.12 ( [see issue](https://github.com/vakata/jstree/issues/2726) )

And wait for fix...

**Reply by Tipetan** [reply]
Hi @Sega100500 
thank you for the answer. I have tried the code but the downgrade has no effect to this problem in my project.
I must wait for a fix.
